inc/dbmysql.class.php

```diff
@@ -714,4 +714,52 @@ static public function checkForCrashedTables() {
       }
       return $crashed_tables;
    }
+
+   /**
+   * Verify if a DB table exists
+   *
+   * @param string $tablename Name of the table we want to verify.
+   *
+   *  @return boolean
+   **/
+   public function tableExists($tablename) {
+      // Get a list of tables contained within the database.
+      $result = $this->list_tables("%".$tablename."%");
+
+      if ($this->numrows($result)) {
+         while ($data = $this->fetch_row($result)) {
+            if ($data[0] === $tablename) {
+               return true;
+            }
+         }
+      }
+
+      $this->free_result($result);
+      return false;
+   }
+
+   /**
+   * Verify if a DB field exists
+   *
+   * @param string  $table    Name of the table we want to verify.
+   * @param string  $field    Name of the field we want to verify.
+   * @param Boolean $usecache If use field list cache (default true)
```

Comment:
i think it's a copypasta of the old method (so not your fault) but this doc line is pretty abstruse